wiki_article_count.py

```diff
@@ -0,0 +1,37 @@
+"""
+Provide a map from language name to Wikipedia article count.
+
+>>> get_language_sizes()
+{
+	"aa": 124,
+	"af": 5995,
+	...
+}
+"""
+
+import csv
+from file_cached import cached
+from io import StringIO
+import re
+import requests
+
+
+url = "https://gitlab.com/wmde/technical-wishes/wiki-article-counter/-/raw/main/wiki_article_counts.csv"
+
+
+@cached("wiki_article_count.csv")
+def get_wiki_counts():
```

Comment:
Interesting example of how to get the article count by Wikipedia language edition.

Reply:
Now I'm leaning towards a port of the other repository, so that the library here is capable of regenerating fresh statistics.

Reply:
(done and pushed to branch)